### PR TITLE
Add kubernetes.core as dependency

### PR DESCRIPTION
**Pull request by sshnaidm**
Use 2.3.2 fixed version to prevent possible breakages

---

TestBos2: virt

### CODE REVIEW

File: galaxy.yml [hunk 1]
```diff
@@ -103,6 +103,7 @@ dependencies:
   "community.kubernetes": "1.2.1"
   "community.libvirt": "*"
   "containers.podman": "*"
+  "kubernetes.core": "3.0.0"
```
Comment (tonyskapunk) [inline galaxy.yml:106]: This should match the RPM package version available (currently [2.3.2](https://softwarefactory-project.io/r/plugins/gitiles/ansible-collection-kubernetes-core)) so both installation methods (RPM and collections) use the same versions

File: ansible-collection-redhatci-ocp.spec
Comment (tonyskapunk) [inline ansible-collection-redhatci-ocp.spec:1]: Let's bump the RPM version.

**Review by tonyskapunk — CHANGES_REQUESTED** [review]
There's some related work in #214, but I like that we can make one step at a time and  move closer to kubernetes core in here.

File: galaxy.yml (hunk 1 above)
Comment (sshnaidm) [reply galaxy.yml:106]: Ok, let's use 2.3.2 then. Seems like latest Ansible binary depends on kubernetes.core < 3, and it can't be updated.

**Comment by tonyskapunk** [comment]
The [dev#pull-requests](https://github.com/redhatci/ansible-collection-redhatci-ocp/blob/main/docs/dev.md#pull-requests) doc contains information on how to test accordingly the changes, in this case it should be enough to validate that the change  is not breaking anything :).

recheck

**Comment by tonyskapunk** [comment]
recheck

**Comment by sshnaidm** [comment]
@tonyskapunk is there anything still blocking this?

**Comment by tonyskapunk** [comment]
> @tonyskapunk is there anything still blocking this?

Yes, it requires a bump to the RPM and the collections, more than happy to help if needed.

**Comment by sshnaidm** [comment]
@tonyskapunk sorry, I'm not sure what you mean. What is bumping RPM and collection?

**Comment by tonyskapunk** [comment]
Ah, my bad, sorry for not being clear,  please increase the minor version for the RPM spec and the galaxy config file:

https://github.com/redhatci/ansible-collection-redhatci-ocp/blob/9547bf3aba2e10569aa8e17b096c1ee6bd111590/galaxy.yml#L12

and

https://github.com/redhatci/ansible-collection-redhatci-ocp/blob/9547bf3aba2e10569aa8e17b096c1ee6bd111590/ansible-collection-redhatci-ocp.spec#L6

**Pull request closed without merging**